_rules/SC1-1-1-accessible-name-is-image-filename.md

```diff
@@ -0,0 +1,88 @@
+---
+name: Image accessible name is filename
+description: |
+  This rule checks that image elements that use their source filename as their accessible name do so without loss of infomation to the user.
+
+success_criterion:
+- 1.1.1 # Non-Text Content
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to HTML `input` element with a [`type`](https://www.w3.org/TR/html/sec-forms.html#dom-htmlinputelement-type) of `image`, or `area` elements, or any [non-decorative](#decorative) HTML element with the semantic role of `img`, where the element is [included in the accessibility tree](#included-in-the-accessibility-tree) and has an [accessible name](#accessible-name) that is equivalent to the [filename](#filename) specified in the `src` attribute. Difference in letter casing, and forward and trailing whitespace should be ignored. 
+
+### Expectation
+
+Each test target has an accessible name that serves an equivalent purpose can of the [non-text content](https://www.w3.org/TR/WCAG21/#dfn-non-text-content).
```

Comment:
The word "equivalent" by itself in ambiguous. It can be interpreted as "each test target has an accessible name that **at least** serves an equivalent purpose of the non-text-content" or it can be interpreted as "each test target has an accessible name that serves **only** an equivalent purpose of the non-text-content".

I suggest the latter.